pages/glossary/programatically-determined-link-context.md

```diff
@@ -0,0 +1,19 @@
+--
+title: Programmatically Determined Link Context
+key: programmatically-determined-link-context
+--
+
+In HTML, information that is programmatically determinable from a link includes [text nodes][] that are [included in the accessibility tree][] with a non-empty ("") [accessible name][] and are
+
+- in the same paragraph (HTML `p` element) as the link;
```

Comment:
Need to clarify what "in the same" means. I suspect what you mean is that it has the same `p` element as an ancestor in the flattened tree, correct?